make/test/BuildMicrobenchmark.gmk

```diff
@@ -120,6 +120,11 @@ $(JMH_UNPACKED_JARS_DONE): $(JMH_RUNTIME_JARS)
             $$($(UNZIP) -oq $(jar) -d $(JMH_UNPACKED_DIR)))
 	$(RM) -r $(JMH_UNPACKED_DIR)/META-INF
 	$(RM) $(JMH_UNPACKED_DIR)/*.xml
+	$(MKDIR) -p $(MICROBENCHMARK_CLASSES)/org/openjdk/bench/javax/xml
+	$(CP) $(TOPDIR)/test/jaxp/javax/xml/jaxp/unittest/validation/tck/reZ003vExc23082309.xml \
+            $(TOPDIR)/test/jaxp/javax/xml/jaxp/unittest/stream/XMLStreamWriterTest/message_12.xml \
+            $(TOPDIR)/test/jaxp/javax/xml/jaxp/unittest/transform/msgAttach.xml \
+            $(MICROBENCHMARK_CLASSES)/org/openjdk/bench/javax/xml
```

Comment:
If this is the correct solution, then copying these files should be handled separately from unpacking jars, preferably using a SetupCopyFiles macro call. Something like this (untested):
```
$(eval $(call SetupCopyFiles, COPY_JAXP_TEST_XML, \
    DEST := $(MICROBENCHMARK_CLASSES)/org/openjdk/bench/javax/xml, \
    FILES := \
        $(TOPDIR)/test/jaxp/javax/xml/jaxp/unittest/validation/tck/reZ003vExc23082309.xml \
        $(TOPDIR)/test/jaxp/javax/xml/jaxp/unittest/stream/XMLStreamWriterTest/message_12.xml \
        $(TOPDIR)/test/jaxp/javax/xml/jaxp/unittest/transform/msgAttach.xml, \
    FLATTEN := true, \
))
```
And then add $(COPY_JAXP_TEST_XML) to the list of dependencies in the `SetupJarArchive` call below.

Reply:
The `SetupCopyFiles` macro seems do not support multiple absolute path `FILES` arguments, so I use relative path instead.